solutions/banking/main.tf

```diff
@@ -245,6 +245,17 @@ resource "ibm_cd_tekton_pipeline_property" "watsonx_assistant_id_pipeline_proper
   value       = var.watson_assistant_instance_id
 }
 
+# Update CD pipeline with Resource Group
+resource "ibm_cd_tekton_pipeline_property" "resource_group_name_pipeline_property_cd" {
+  depends_on  = [local.cd_instance]
+  count       = var.resource_group_name != null ? 1 : 0
+  provider    = ibm.ibm_resources
+  name        = "dev-resource-group"
+  pipeline_id = var.ci_pipeline_id
```

Comment:
@akocbek @brendankellyibm @ocofaigh - if this is for CD pipeline - why is it going into var.ci_pipeline_id ?